### PR TITLE
Add goToUrl action creator

### LINKED ISSUE #30: Support navigateByUrl equivalent
There are times when `go()` is not sufficient as it uses `navigate` method internally. For example, when passing redirection links with URL params inside, internal use of `navigate` doesn't work well and params are not parsed properly. This can be solved by using `navigateByUrl`.

E.g. passing continue link with parameters: `/candidates/57f39a88720e1a181134ae25;back=%2Fcandidates`

This doesn't work well when doing `go('/candidates/57f39a88720e1a181134ae25;back=%2Fcandidates')`. The `back` parameter is not parsed properly and angular2 thinks it's actually the `:id` parameter in route definition (`/candidates/:id`). Instead, I have to use `router.navigateByUrl('/57f39a88720e1a181134ae25;back=%2Fcandidates')` which ensures the parsing is being done.

### PR DESCRIPTION
**Pull request by mischkl**
Fixes #30

### CODE REVIEW

File: src/actions.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ export interface RouterMethodCall {
 
 export const routerActions = {
   GO: '[Router] Go',
+  GO_TO_URL: '[Router] Go to URL',
```
Comment (brandonroberts) [inline src/actions.ts:12]: Change the action to `URL` instead

File: src/actions.ts [hunk 2]
```diff
@@ -25,6 +26,12 @@ export function go(path: string|any[], query?: any, extras?: NavigationExtras):
   return { type: routerActions.GO, payload };
 }
 
+export function goToUrl(path: string, extras?: NavigationExtras): Action {
```
Comment (brandonroberts) [inline src/actions.ts:29]: Change the function name to `url`

File: src/connect.ts [hunk 3]
```diff
@@ -22,6 +22,12 @@ export function listenForRouterMethodActions(router: Router, location: Location,
           router.navigate(commands, Object.assign({}, extras, { queryParams }));
           break;
 
+        case routerActions.GO_TO_URL:
+          if (typeof path === 'string') {
```
Comment (brandonroberts) [inline src/connect.ts:26]: The action is already typed to a `string` so no need to validate it here.

**Review by brandonroberts — CHANGES_REQUESTED** [review]

**Comment by mischkl** [comment]
Hello, may I ask why this was closed? Is this feature going to be implemented still?

**Comment by jinder** [comment]
@mischkl I think because they're concentrating on NGRX 4, which will use the normal router methods not dispatchable actions.

**Pull request closed without merging**